packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -119,11 +112,12 @@ abstract contract CrossDomainMessenger is
     ///         can therefore not be relayed again.
     mapping(bytes32 => bool) public successfulMessages;
 
-    /// @notice Address of the sender of the currently executing message on the other chain. If the
-    ///         value of this variable is the default value (0x00000000...dead) then no message is
-    ///         currently being executed. Use the xDomainMessageSender getter which will throw an
-    ///         error if this is the case.
-    address internal xDomainMsgSender;
+    /// @custom:legacy
+    /// @custom:spacer xDomainMsgSender
+    /// @notice Spacer for backwards compatibility. The storage slot was migrated when the
+    ///         initializer pattern was moved away from in the base contract to remove the
+    ///         need to set `Constants.DEFAULT_L2_SENDER` into storage during a call to `initialize`.
+    bytes20 internal spacer_204_0_20;
```

Comment:
it's not clear to me why this needed to move. The initialized slot comes afterwards.

Reply:
I grok the comment now, and also that this removes the need to zero out the existing value in the L2 contract.